gameofgame.py

```diff
@@ -25,49 +25,65 @@ def choose_country():
 # Function to check if the guess is correct
 def check_guess(guess, country):
     capital = countries_capitals[country]
-
     if guess.lower() == capital.lower():
-    return True
+        return True
     else:
-    return False
- main
+        return False
+
+# Initialize player scores
+player1_score = 0
+player2_score = 0
 
 # Main game loop
-    while True:
+while True:
     # Choose a random country
     country = choose_country()
 
     # Both Players take turns to play
-    #player1 goes first
+    # player 1 goes first
     print("Player 1's turn")
     player1_guess = input(f"What is the capital of {country}? ")
```

Comment:
The text that is printed gets repeated between player 1 & player 2. An option to simplify (i.e. avoid duplication) the code would be to define variable with the text (as a string) and use that one instead of duplicating the message that is being printed.